helloworld/src/App.css

```diff
@@ -0,0 +1,37 @@
+.App {
```

Comment:
Check file css here, in App.js file I saw that you didn't use that. Also naming class name is not good, shouldn't capitalize